internal/kafka/internal/services/kafkatlscertmgmt/kafka_tls_certificate_management_service.go

```diff
@@ -147,9 +152,18 @@ func (certManagementService *kafkaTLSCertificateManagementService) RevokeCertifi
 		return nil // the certificate is revoked manually in manual mode
 	}
 
+	wildcardDomain := fmt.Sprintf("*.%s", domain)
+	refs := certManagementService.certManagementClient.GetCerticateRefs(wildcardDomain)
+
+	// No need to revoke the certificate if it does not exists
+	if !certManagementService.storage.Exists(ctx, refs.TLSCertRef) {
+		logger.NewUHCLogger(ctx).V(10).Infof("certificate for domain %q does not exist in Storage. It is not going to be revoked", domain)
```

Comment:
do we need to create a new logger each time?

Reply:
Yes, as the `ctx Context` object is a different. 
That being said, we could use the singleton logger that we've.